scripts/benchmark/README.md

```diff
@@ -0,0 +1,33 @@
+Script to run an executable (for instance JBMC) on a list of Java methods.
+
+# Install
+
+    npm install
+
+# Usage
+
+    benchmark_java_project.js --help
+
+# Example
+
+From `apache-tika/tika-core/target/classes` run:
+
+    ./benchmark_java_project.js /path/to/jbmc -l method_list_example.txt -a java_arguments.json -m path/to/java-models-library/target/core-models.jar | tee result.json
```

Comment:
Since you are including `java_arguments.json`, would you mind adding `method_list_example.txt`?